recipes/cms/meta.yaml

```diff
@@ -0,0 +1,119 @@
+package:
+  name: cms
+  version: "2.0.0b"
+
+about:
+  home: http://broad-cms.readthedocs.io/en/latest/
+  license: GPLv3
+  summary: "CMS (composite of multiple signals) is a suite of tools for resolving signatures of recent positive selection in population genetic data."
+
+source:
+  git_url: https://github.com/broadinstitute/cms.git
+  git_rev: ee7efbb5b18697fc9bf2e813fe76fbbc2439a0de
+  
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc   # [not osx]
```

Comment:
zlib seems to be missing, for more information see: https://github.com/bioconda/bioconda-recipes/blob/r29/GUIDELINES.md#zlib